libs/langchain/langchain/schema/runnable/base.py

```diff
@@ -117,6 +172,7 @@ def input_schema(self) -> Type[BaseModel]:
 
     @property
     def output_schema(self) -> Type[BaseModel]:
+        """The type of output this runnable produces specified as a pydantic model."""
```

Comment:
Add here something like "to mark a field as configurable see the methods configurable_...